### DISCUSSION TITLE
Box as time is missing datetime property

### DISCUSSION THREAD

**Discussion by tomtobac** [twilio-labs/paste#661] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/10562610/90501904-58658100-e14d-11ea-9ddc-493467787473.png)

It should allow to use datetime attribute
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

**Comment by corinne-nagel** [comment, marked as answer]
Hi @tomtobac reaching out to the team now, should have a response soon. Thanks!

**Reply by corinne-nagel** [reply]
@tomtobac I've created a ticket in our backlog; we will let you know when this has been added, thanks!
https://issues.corp.twilio.com/browse/DSYS-2177

**Reply by TheSisb** [reply]
@tomtobac Fixed in https://github.com/twilio-labs/paste/pull/1253
